UI/window-basic-settings-stream.cpp

```diff
@@ -547,13 +630,37 @@ void OBSBasicSettings::on_customServer_textChanged(const QString &)
 
 	protocol = FindProtocol();
 	UpdateAdvNetworkGroup();
+	UpdateMultitrackVideo();
 
 	if (ServiceSupportsCodecCheck())
 		lastCustomServer = ui->customServer->text();
 
 	SwapMultiTrack(QT_TO_UTF8(protocol));
 }
 
+void OBSBasicSettings::on_enableMultitrackVideo_toggled(bool /*enabled*/)
+{
+	UpdateMultitrackVideo();
+}
+
+void OBSBasicSettings::on_multitrackVideoMaximumAggregateBitrateAuto_toggled(
+	bool /*enabled*/)
+{
+	UpdateMultitrackVideo();
+}
+
+void OBSBasicSettings::on_multitrackVideoMaximumVideoTracksAuto_toggled(
+	bool /*enabled*/)
+{
+	UpdateMultitrackVideo();
+}
+
+void OBSBasicSettings::on_multitrackVideoConfigOverrideEnable_toggled(
+	bool /*enabled*/)
+{
+	UpdateMultitrackVideo();
+}
+
```

Comment:
Turn `UpdateMultitrackVideo` into a slot and connect to widgets.

Reply:
changed right after https://github.com/obsproject/obs-studio/pull/10633#event-12771455631

Reply:
Can be put as resolved.